### ISSUE TITLE
Fix pre-authenticated buttons in Safari (iOS)

### ISSUE THREAD

**Issue by LukasKalbertodt** [elan-ev/tobira#831]
The buttons for Studio and editor don't work if pre-authentication is enabled. In that case, it's actually a `<form>` with some hidden inputs. And the button is supposed to submit that form. That doesn't happen on Safari though (at least on iOS). Maybe because we have a custom click handler attached to the button, the default "submit" behavior isn't triggered anymore?

**Comment by LukasKalbertodt** [comment]
I did turn off "Content blockers" and my "AdGuard" extension, but still: doesn't work.

**Comment by owi92** [comment]
Yeah I was afraid that it wouldn't help either. This is a pain to debug.
